examples/QmlBarcodeGenerator/cmake/Locations.cmake

```diff
@@ -3,23 +3,21 @@ set(COPYRIGHT "Copyright (c) 2022 Scythe Studio. All rights reserved.")
 set(IDENTIFIER "com.scythestudio.scodes.example")
 
 # ---CONFIGURATION---
-# MOST IMPORTANT PART -> change to ON/OFF if you want to use these elements in your project
 option(USE_QML "Add QML support" ON)
 option(USE_LIBS "Use external libraries" ON)
 
-set(QT_NAME Qt5)
-
 # Locations - directories in project structure
-set(LIB_DIR "../../../src")
+set(LIB_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../../src)
```

Comment:
Instead of mixing relative paths and variables you can use `CMAKE_SOURCE_DIR` variable - this points to directory containing the root CMakeLists.txt, which in that case is a directory of `QmlBarcodeGenerator`.